src/components/StopWatch.tsx

```diff
@@ -1,26 +1,33 @@
 import { useEffect, useState } from "react";
+import { Button } from "./Button";
 import { Timer } from "./Timer";
 export const StopWatch = () => {
   const [time, setTime] = useState(0);
-  const [running, setRunning] = useState(false);
+  const [status, setStatus] = useState("");
```

Comment:
if you want to use the "status" state in this way, it's better to use the string literals type instead of the string type

```tsx
useState<"paused" | "started" | "reset">("paused")
```

Although I think the previous Boolean state was better "running" because it's clearer